lib/wings/services/custom_queries/find_many_by_alternate_ids.rb

```diff
@@ -22,7 +22,7 @@ def initialize(query_service:)
       # @param alternate_ids [Enumerator<#to_s>] list of ids
       # @param use_valkyrie [boolean] defaults to true; optionally return ActiveFedora::Base objects if false
       # @return [Array<Valkyrie::Resource>, Array<ActiveFedora::Base>]
-      def find_many_by_alternate_ids(alternate_ids:, use_valkyrie: true)
+      def find_many_by_alternate_ids(alternate_ids:, use_valkyrie: Hyrax.config.use_valkryie?)
```

Comment:
i think these checks in `wings` should keep their existing defaults.

in truth, these should be deprecated. AFAIK, there's not a good use case for not using valkyrie within wings.

Reply:
Yep, I talked with @elrayle about it. I reverted them back to the hardcoded defaults.